web/themes/custom/itkdev/itkdev_project_theme/templates/node--project.html.twig

```diff
@@ -88,7 +88,7 @@
             <div class="mb-3">{{ projectTrack.getDescription()|raw }}</div>
             <div class="flex justify-between">
               <div class="flex justify-start">
-                {% if projectTrack.getType.field_report_type.value is same as('bubble_chart') %}
+                {% if projectTrack.getType.field_report_type.value is same as('bubble_chart') or projectTrack.getType.field_report_type.value is same as('webform_submission') %}
```

Comment:
Use [`in`](https://twig.symfony.com/doc/3.x/templates.html#containment-operators) here:

```suggestion
                {% if projectTrack.getType.field_report_type.value in ['bubble_chart', 'webform_submission']) %}
```